### DISCUSSION TITLE
Missing wEth in warmhole

### DISCUSSION THREAD

**Discussion by RafaelFernandez** [wormhole-foundation/wormhole#432] [Problems / bugs] [answered]
Please help. I used wormhole to transfer wEth from Ethereum  to Solana but I did not receive them. I can not see the transaction in wormholebridge.com. How can I get back my wEth in Ethereum or in Solana?
Ethereum transaction: [https://etherscan.io/tx/0xb9fef33493c1f485a50c29a54a937f19d02728e580095ad3cd53c6e38b418575]
Destination address in Solana: 2wWxCKJpdrG1n3puG6g3D2ugj5n6CgZELcWHzfTHaEMZ
Thanks in advance.

**Comment by leoluk** [comment, marked as answer]
Duplicate of https://github.com/solana-labs/oyster/issues/318
